src/bundle/Resources/config/services/forms.yml

```diff
@@ -47,6 +47,10 @@ services:
         arguments:
             $siteAccessLanguages: '$languages$'
 
+    EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader\ConfiguredLanguagesChoiceLoader:
+        arguments:
+            $siteAccessLanguages: '$languages$'
```

Comment:
This is risky and currently does not work in 3.0, please inject ConfigResolver instead.